src/runtime/node/module/$cloudflare.ts

```diff
@@ -0,0 +1,88 @@
+// https://nodejs.org/api/module.html
+import type nodeModule from "node:module";
+
+export {
+  Module,
+  SourceMap,
+  _cache,
+  _extensions,
+  _debug,
+  _pathCache,
+  _findPath,
+  _initPaths,
+  _load,
+  _nodeModulePaths,
+  _preloadModules,
+  _resolveFilename,
+  _resolveLookupPaths,
+} from "./index";
+
+import {
+  Module,
+  SourceMap,
+  _cache,
+  _extensions,
+  _debug,
+  _pathCache,
+  _findPath,
+  _initPaths,
+  _load,
+  _nodeModulePaths,
+  _preloadModules,
+  _resolveFilename,
+  _resolveLookupPaths,
+  builtinModules as unenvBuiltinModules,
+  createRequire as unenvCreateRequire,
+  findSourceMap as unenvFindSourceMap,
+  globalPaths as unenvGlobalPaths,
+  isBuiltin as unenvIsBuiltin,
+  register as unenvRegister,
+  runMain as unenvRunMain,
+  syncBuiltinESMExports as unenvSyncBuiltinESMExports,
+  wrap as unenvWrap,
+} from "./index";
+
+const workerdModule = process.getBuiltinModule("node:module");
```

Comment:
@IgorMinar has to confirm but i guess `process` used to be injected with polyfill)

Reply:
We use it at https://github.com/unjs/unenv/blob/70db6f157dde54d1b11f4fa77cee630fb255be49/src/runtime/node/buffer/%24cloudflare.ts#L29

Reply:
Indeed which got me suprised because AFAIK it was problematic for workerd in someplace else there are comments about it.

Reply:
https://github.com/unjs/unenv/pull/301#issuecomment-2326151315